mk/crypto.mk

```diff
@@ -1,14 +1,7 @@
 # SPDX-License-Identifier: Apache-2.0
-LDLIBS += -L$(LIB_DIR)
-
-LIBDEPS += $(LIB_DIR)/libfips202.a
-LDLIBS += -lfips202
 CPPFLAGS += -Ifips202 -Ifips202/native
 
-FIPS202_SRCS = $(wildcard fips202/*.c)
+SOURCES += $(wildcard fips202/*.c)
 ifeq ($(OPT),1)
-	FIPS202_SRCS += $(wildcard fips202/native/aarch64/*.S) $(wildcard fips202/native/x86_64/xkcp/*.c)
-	CPPFLAGS += -DMLKEM_USE_NATIVE
```

Comment:
Why was this removed?

Reply:
I simply added the fips202 source code to the `SOURCES` variable, which is then used in `mk/schemes.mk`

And this is merely a quick workaround, I wanted to first see if things worked as expected, then cleanup afterwards

Reply:
I mean why was the change to CPPFLAGS removed?